partcad/src/partcad/project.py

```
@@ -790,13 +834,24 @@ def get_object(
-                config = config_class.normalize(object_name, config, full_object_name)
-                self.init_object_by_config(factory_name, config_class, alias_class, config)
+                try:
+                    config = get_config(object_name)
+                    full_object_name = f"{self.name}:{object_name}"
+                    config = config_class.normalize(object_name, config, full_object_name)
+                    self.init_object_by_config(factory_name, config_class, alias_class, config)
+                except Exception as e:
+                    self.record_broken_object(factory_name, object_name, e)
+                    return None
 
                 if not object_name in objects or objects[object_name] is None:
```

Ruff (0.16.1)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)